src/CAPNet/CAPNet.csproj

```diff
@@ -170,11 +166,7 @@
     <None Include="CAPNet.ruleset">
       <SubType>Designer</SubType>
     </None>
-    <None Include="packages.config" />
-  </ItemGroup>
-  <ItemGroup>
-    <Analyzer Include="..\packages\SonarLint.1.8.0\analyzers\SonarLint.CSharp.dll" />
```

Comment:
I'm a bit sad to see the Roslyn analyzers gone. Although it seems that analyzers are finally coming to .NET Core: https://blogs.msdn.microsoft.com/devfish/2017/11/09/rigging-up-roslyn-analyzers-in-net-core/